integration-tests/jest.config.ts

```diff
@@ -3,8 +3,6 @@ import baseConfig from "../jest.config.base";
 
 export default {
   ...baseConfig,
-  projects: ["tests/*/jest.config.ts"],
-  testMatch: ["<rootDir>/**/*.test.ts"],
-  collectCoverage: false,
+  testMatch: ["<rootDir>/tests/infra/**/*.test.ts"], // Adjust to specifically match infra tests
```

Comment:
Do we not run any other tests from the integration-test folder at the moment?

Reply:
Apologies, I removed these when debugging issues with the jest.config. I've reverted these changes now